banks-interface/src/lib.rs

```diff
@@ -10,11 +10,19 @@ use solana_sdk::{
     transaction::{self, Transaction, TransactionError},
 };
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum TransactionConfirmationStatus {
+    Unconfirmed,
+    OptimisticallyConfirmed,
```

Comment:
Am I being to optimistic here?
```suggestion
    Optimistic,
```

Reply:
🤣 
Does make sense, though, with the new field name.